doc/radosgw/multisite.rst

```diff
@@ -551,7 +551,7 @@ Starting the radosgw
 --------------------
 
 On the node that is hosting the radosgw of the secondary zone, you would start
-it, similar to the `Starting the Radosgw`_, changing the ``rgw zone=us-east-2``
+it, similar to the `Starting the radosgw`_, changing the ``rgw zone=us-east-2``
```

Comment:
I thought he was going to referencing https://github.com/ceph/ceph/pull/8660/files#diff-592268cf2187d55ebaf13e62663ea80bR394 but not very sure this is the right fix.


> might want to add an anchor at https://github.com/ceph/ceph/pull/8660/files#diff-592268cf2187d55ebaf13e62663ea80bR394 if the reference is wrong.

Actually I have no idea how to anchor this as there seems to be two sections of the same title "Starting the radosgw" here:(